packages/devtools_app/lib/src/shared/primitives/encoding.dart

```diff
@@ -88,6 +88,10 @@ Object? toEncodable(Object? value) {
       HeapSnapshotGraphEncodeDecode.instance.toEncodable(value),
     final ByteData value => ByteDataEncodeDecode.instance.toEncodable(value),
     final DateTime value => DateTimeEncodeDecode.instance.toEncodable(value),
-    _ => value,
+    _ =>
+      // For some reasons the failures show different error:
+      // `Converting object to an encodable object failed: Instance of 'some other type'`.
+      // To see the actual type, put breakpoint here:
+      throw StateError('Unsupported type: ${value.runtimeType}'),
```

Comment:
do we still want to throw an exception at runtime? Or should we log the error and continue gracefully?

Reply:
If we got to this line, it will fail anyway.
And actually UI handles it gracefully. Save just does not happen.